packages/xo-web/src/xo-app/backup/health/index.js

```diff
@@ -54,28 +48,25 @@ const DETACHED_BACKUP_COLUMNS = [
   {
     name: _('vm'),
     itemRenderer: ({ vmId }) => <Vm id={vmId} link />,
-    sortCriteria: ({ vmId }, { vms }) => {
-      const vm = vms[vmId]
-      return vm !== undefined && vm.name_label
-    },
+    sortCriteria: ({ vmId }, { vms }) => get(() => vms[vmId].name_label),
   },
   {
     name: _('job'),
     itemRenderer: ({ jobId }, { jobs }) => {
       const job = jobs[jobId]
-      return job === undefined ? null : (
+      return job === undefined ? (
+        <Copiable data={jobId} tagName='p'>
```

Comment:
@badrAZ  What do you think about this and sorting?

Reply:
I would say to have a job render XO item instead.